src/renderer/component/common.js

```diff
@@ -1,25 +1,11 @@
+// just disabling the linter because this file shouldn't even exist
+// will continue to move components over to /components/common/{comp} - sean
+/* eslint-disable */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { formatCredits, formatFullPrice } from 'util/formatCredits';
 import lbry from '../lbry.js';
 
```

Comment:
Great call to refactor this file and eliminate this "common" bullshit :)

Any reason not to just make these normal / typical first-class components?

Reply:
I have been thinking of `/common/*` simple utility components. Then we can avoid the all of the `connect(null, null)(Component)` index.js files which a few components have.